README.md

```diff
@@ -175,6 +175,8 @@ with the pep8 style.  We ask that before you submit a pull request, you:
 
 PEP8: https://www.python.org/dev/peps/pep-0008/
 
+You can install development dependencies with `pip install -e .[dev]`
```

Comment:
This is great!  Would you mind also adding this note to the Makefile, which has some code that detects if flake8 isn't installed and instructs you to download it in an error message?  That seems like another good oppurtunity to point people at this new feature.

Reply:
Added the requested message, but was messing around and did some formatting to make the messages look better, hopefully that's okay too. 

![screen shot 2018-06-20 at 8 53 18 pm](https://user-images.githubusercontent.com/3489076/41697407-7b432d06-74cd-11e8-90cc-d3be537a6ef2.png)
![screen shot 2018-06-20 at 8 52 28 pm](https://user-images.githubusercontent.com/3489076/41697408-7b584baa-74cd-11e8-8a14-62d581053106.png)